src/main/java/org/fundacion/jala/converter/view/metadata/SelectFile.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package org.fundacion.jala.converter.view.metadata;
+
+import javax.swing.JPanel;
+import javax.swing.JButton;
+import javax.swing.JTextField;
+import javax.swing.JFileChooser;
+
+import java.awt.Dimension;
+import java.awt.FlowLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.File;
+
+class SelectFile extends JPanel implements ActionListener {
+    private JButton selectFile;
+    private JTextField textFile;
+    private JFileChooser fileChooser;
+    private String fileOriginPath;
+
+    /**
+     * Initialize the graphic components for Selecting file display.
+     */
+    protected SelectFile() {
+        selectFile = new JButton("Open File");
+        selectFile.addActionListener(this);
+        int textFieldHeight = 30;
+        int textFieldWidth = 350;
+        textFile = new JTextField();
+        textFile.setPreferredSize(new Dimension(textFieldWidth, textFieldHeight));
+        textFile.setEnabled(false);
+
+        int marginHeight = 30;
```

Comment:
Magic numbers, could be set as final attributes.

Reply:
Thanks for suggestion, it is solved now :)